src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml

```diff
@@ -77,7 +77,7 @@ Column {
             propertyItem: root.dashLineLength
 
             step: 0.1
-            maxValue: 10
+            maxValue: 100
```

Comment:
@bkunda 100 is not too much here?

Reply:
also it might make sense to increase a value step then

Reply:
It depends for how many measures it lasts. I am a beginner in writing scores. If it lasts for 10 measures, then even a value of 100 makes sense.

Reply:
100 is pretty excessive in most cases, but I'm not entirely against it because it does give the user quite a lot of control over how their dashes look, and in some extreme cases (lines extending over many systems, for example), it could actually be somewhat useful. 

I also think it best to keep the step value at 0.1 at the moment, just because this behaves quite differently depending on the line length itself (0.1. causes barely perceptible results for changes in increments <0.4 in short lines, but produces more obvious changes in increments of 0.1 for long lines stretching the length of an entire system). 

As a general point though, I think having the option to use `Shift`&#8593; and `Shift`&#8595; to increase all text fields by step value of, say, 10 could be a useful feature, but this is perhaps the subject of a different issue.